### PR TITLE
Add documentation for flatpak

### PR DESCRIPTION
**Pull request by perlpunk**

### CODE REVIEW

File: xml/obs_package_formats.xml [hunk 1]
```diff
@@ -282,4 +282,124 @@ systemctl enable sshd</screen>
...
  </sect1>
+
+ <sect1>
+  <title>&flatpak;</title>
+  <para>
+  The <link xlink:href="https://flatpak.org/">Flatpak</link> format can be used
+  to generate desktop apps for linux.
+  </para>
+  <para>
+  For building an installable flatpak bundle, you need a
```
Comment (tomschr) [inline xml/obs_package_formats.xml:293]: We should capitalize project names. Look for other occurances. :slightly_smiling_face: 

```suggestion
  For building an installable Flatpak bundle, you need a
```

File: xml/obs_package_formats.xml [hunk 2]
```diff
@@ -282,4 +282,124 @@ systemctl enable sshd</screen>
...
+  <filename>flatpak.yaml</filename> manifest.
```
Comment (tomschr) [inline xml/obs_package_formats.xml:294]: I would be explicit here:

```suggestion
  <filename>flatpak.yaml</filename> manifest file.
```

File: xml/obs_package_formats.xml [hunk 3]
```diff
@@ -282,4 +282,124 @@ systemctl enable sshd</screen>
...
+  <listitem><para>
+  For a quick start, branch the
+  <link xlink:href="https://build.opensuse.org/package/show/OBS:Flatpak:Templates/FlatpakTemplate">Template Package</link>.
+  </para></listitem>
+  <listitem><para>
+  To learn YAML, have a look at this <link xlink:href="https://www.yaml.info/">YAML Tutorial</link>.
+  </para></listitem>
+  </itemizedlist>
+  <example>
+   <title>&flatpak; Manifest</title>
```
Comment (tomschr) [inline xml/obs_package_formats.xml:332]: I would give this example a filename.

File: xml/obs_package_formats.xml [hunk 4]
```diff
@@ -282,4 +282,124 @@ systemctl enable sshd</screen>
...
+  You can use JSON as it is a subset of YAML. But be aware that flatpak
+  accepts non-standard <literal>//</literal> comments, while &obs; does not.
+  </para></listitem>
+  <listitem><para>
+  <literal>#!BuildVersion</literal> - Use this field to specify the version
+  of your app so the <literal>.flatpak</literal> file will be versioned.
+  Flatpak manifests do not have a version field.
+  </para></listitem>
+  <listitem><para>
+  The base images (freedesktop, gnome, kde) are maintained in the
```
Comment (tomschr) [inline xml/obs_package_formats.xml:317]: I would remove the parenthesis and use the project names. As an alternative, you can keep the parenthesis, but use a tag around the image names, for example, with `literal`: `<literal>kde</literal>`.

```suggestion
  The base images for Freedesktop, GNOME, or KDE are maintained in the
```

**Review by tomschr — CHANGES_REQUESTED** [review]
Thanks Tina for your pull request, much appreciated! :+1: 

I've left some suggestions (see below). They are mostly minor things (structure, spelling etc). Hope it helps. Let me know if you have any questions.

**Comment by perlpunk** [comment]
Thanks @tomschr !
I think I addressed all of your suggestions :)

**Comment by perlpunk** [comment]
Is there anything else I can do? :)

**Comment by perlpunk** [comment]
@eduardoj made some suggestions, so I did a tiny restructure of the first paragraphs.

**Comment by perlpunk** [comment]
When will this go live? Is there some automated deployment?
Still can't see it here https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.package_formats.html

**Comment by adrianschroeter** [comment]
On Freitag, 19. Februar 2021, 10:52:36 CET Tina Müller (tinita) wrote:
> When will this go live? Is there some automated deployment?
> Still can't see it here https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.package_formats.html

it is live

-- 

Adrian Schroeter <adrian@suse.de>
Build Infrastructure Project Manager

SUSE Software Solutions Germany GmbH,  Maxfeldstr. 5, 90409 Nuernberg, Germany
(HRB 247165, AG München), Geschäftsführer: Felix Imendörffer

**Comment by perlpunk** [comment]
Hm, then I wonder why I can't see the new entry. Did I miss something in my PR?

**Comment by adrianschroeter** [comment]
On Freitag, 19. Februar 2021, 11:05:25 CET Tina Müller (tinita) wrote:
> Hm, then I wonder why I can't see the new entry. Did I miss something in my PR?

ah, sorry, mixed it up with your implementation in OBS code.

I will push new build of the documentation in next minutes ...

-- 

Adrian Schroeter <adrian@suse.de>
Build Infrastructure Project Manager

SUSE Software Solutions Germany GmbH,  Maxfeldstr. 5, 90409 Nuernberg, Germany
(HRB 247165, AG München), Geschäftsführer: Felix Imendörffer

**Pull request merged**